ingestion/src/levels.rs

```diff
@@ -109,10 +140,15 @@ pub async fn fetch_levels(stinfo_conn_string: &str) -> Result<ParamLevelTable, E
     });
 
     // query param table
+    // TODO: should we care about cases where standard_hlevel is NULL,
+    // while hlevel_scale is NOT NULL?
+    // Right now, for this case, we insert NULL level for every incoming level
```

Comment:
We changed this to insert 0 instead of NULL if the incoming level is 0. Still unsure if it's the way to go 🤔

Reply:
Are we ok with this?

Reply:
I guess? We can remove the todo I think